plugins/SupportEraser/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2015 Ultimaker B.V.
```

Comment:
It's 2018, welcome to the future! :)

Reply:
Good point. I copied `__init__.py` from the original Mirror tool.

[Personally I'm not a fan of adding this kind of information into code](https://hackerboss.com/get-rid-of-templates/), partially for this reason, but since I guess Ultimaker has its standards:

Should these files (including the `SupportEraser.py` file where I forgot it) read "2018"? Or does Ultimaker do some kind of "2015-2018"? Not sure if it's per file/per plugin/or should match for everything which is distributed in the Cura application.

Reply:
I don't know. Someone started it, i just copied it. I tend to just use the year in which i wrote it.